File: src/main/java/net/ndrei/teslacorelib/inventory/ISerializableFluidTank.java
package net.ndrei.teslacorelib.inventory;

import net.minecraft.nbt.NBTTagCompound;
import net.minecraftforge.common.util.INBTSerializable;
import net.minecraftforge.fluids.IFluidTank;

/**
 * Created by CF on 2016-12-29.
 */
public interface ISerializableFluidTank extends IFluidTank, INBTSerializable<NBTTagCompound> {
}

File: src/main/kotlin/net/ndrei/teslacorelib/inventory/ISerializableFluidTank.kt
package net.ndrei.teslacorelib.inventory

import net.minecraft.nbt.NBTTagCompound
import net.minecraftforge.common.util.INBTSerializable
import net.minecraftforge.fluids.IFluidTank

/**
 * Created by CF on 2017-06-28.
 */
interface ISerializableFluidTank : IFluidTank, INBTSerializable<NBTTagCompound>
